src/main/java/com/questhelper/helpers/quests/atfirstlight/AtFirstLight.java

```diff
@@ -0,0 +1,356 @@
+	}
+
+	private void setupSteps()
+	{
+		talkToApatura = new NpcStep(this, NpcID.GUILDMASTER_APATURA, new WorldPoint(1554, 3033, 0),
+			"Talk to Guildmaster Apatura in the Hunter Guild, south-west of Civitas illa Fortis.");
+		talkToApatura.addDialogSteps("Can I help you with anything?", "Yes.");
+		goDownTree = new ObjectStep(this, ObjectID.STAIRS_51641, new WorldPoint(1557, 3048, 0),
+			"Go down the stairs in the tree in the guild.");
+		talkToVerity = new NpcStep(this, NpcID.GUILD_SCRIBE_VERITY, new WorldPoint(1559, 9464, 0),
+			"Talk to Guild Scribe Verity behind the bar.");
+		talkToWolf = new NpcStep(this, NpcID.GUILD_HUNTER_WOLF_MASTER, new WorldPoint(1555, 9462, 0),
+			"Talk to Guild Hunter Wolf (Master), next to the bar.");
+		windUpToy = new DetailedQuestStep(this, "Wind up a toy mouse.", toyMouse.highlighted());
+		windUpToy.addIcon(ItemID.TOY_MOUSE_WOUND);
+		useToyOnKiko = new NpcStep(this, NpcID.GUILD_HUNTER_KIKO, new WorldPoint(1552, 9460, 0),
+			"Use the wound up toy mouse on Guild Hunter Kiko near the bar.", toyMouseWound.highlighted());
+		checkBed = new ObjectStep(this, ObjectID.CAT_BED, new WorldPoint(1552, 9460, 0),
+			"Search the cat's bed.");
+		returnToWolf = new NpcStep(this, NpcID.GUILD_HUNTER_WOLF_MASTER, new WorldPoint(1555, 9462, 0),
+			"Return to Guild Hunter Wolf (Master), next to the bar.");
+		goUpTree = new ObjectStep(this, ObjectID.STAIRS_51642, new WorldPoint(1557, 9449, 0),
+			"Go back up the stairs.");
+		talkToFox = new NpcStep(this, NpcID.GUILD_HUNTER_FOX, new WorldPoint(1623, 2982, 0),
+			"Talk to Guild Hunter Fox near the crevice south-east of the Hunter Guild.");
+		takeLeaf = new ObjectStep(this, NullObjectID.NULL_50876, new WorldPoint(1618, 2979, 0),
+			"Search the leafy bush south of the crevice for a smooth leaf.");
+		takeSecondLeaf = new ObjectStep(this, NullObjectID.NULL_50877, new WorldPoint(1673, 2992, 0),
+			"Search the rough-looking bush on the west side of the Locus Oasis.");
+		catchJerboa = new DetailedQuestStep(this, new WorldPoint(1664, 3003, 0),
+			"Catch two Embertailed Jerboa for their tails.", boxTrap.highlighted());
+		catchJerboa.addIcon(ItemID.BOX_TRAP);
+		useTailOnLeaves = new DetailedQuestStep(this, "Use the tails on the leaves.",
+			jerboaTail.highlighted(), smoothLeaf.highlighted());
+		returnToFox = new NpcStep(this, NpcID.GUILD_HUNTER_FOX, new WorldPoint(1623, 2982, 0),
+			"Bring the poultice back to Fox.", makeshiftPoultice);
+		talkToFoxAfterPoultice = new NpcStep(this, NpcID.GUILD_HUNTER_FOX, new WorldPoint(1623, 2982, 0),
+			"Talk to Fox.");
+		talkToAtza = new NpcStep(this, NpcID.ATZA, new WorldPoint(1696, 3063, 0),
+			"Talk to Atza in one of the buildings outside Civitas illa Fortis's south wall, west of the general store.",
+			furSample);
+		talkToAtzaAfterHandingFur = new NpcStep(this, NpcID.ATZA, new WorldPoint(1696, 3063, 0),
+			"Talk to Atza in one of the buildings outside Civitas illa Fortis's south wall, west of the general store.");
```

Comment:
nit: spelling? I'm not an english expert but I think this is more correct
```suggestion
			"Talk to Atza in one of the buildings outside Civitas illa Fortis' south wall, west of the general store.",
			furSample);
		talkToAtzaAfterHandingFur = new NpcStep(this, NpcID.ATZA, new WorldPoint(1696, 3063, 0),
			"Talk to Atza in one of the buildings outside Civitas illa Fortis' south wall, west of the general store.");
```